src/components/Button/Button.tsx

```diff
@@ -159,21 +183,26 @@ export default class Button extends React.PureComponent<ButtonProps, {}> {
       );
 
     return (
-      <BaseButton
-        ariaLabel={ariaLabel}
-        className={this.getClasses()}
-        disabled={disabled}
-        href={href}
-        onBlur={onBlur}
-        onClick={onClick}
-        onFocus={onFocus}
-        onMouseEnter={onMouseEnter}
-        onMouseLeave={onMouseLeave}
-      >
+      <span className="y-button--contents">
```

Comment:
you could add a class like `y-button--contents__hidden` if status==='loading' and target that with one CSS selector instead of two

Reply:
True.